docs/tutorials/python/kvstore.md

```diff
@@ -86,7 +86,7 @@ print(a.asnumpy())
 `[[ 4.  4.  4.],[ 4.  4.  4.]]`<!--notebook-skip-line-->
 
 ```python
-kv.push(3, mx.nd.ones(shape))
+kv.push(3, mx.nd.ones(shape, contexts[0]))
```

Comment:
If there are multiple GPUs, will this change instruct user only storing NDArray to the first one?

Reply:
Since this is an user guide page, shouldn't we be more generic to have a simple user experience? I feel it might be better to fix this in nightly pipeline than updating user guide.

Reply:
I think we should make the tutorial simpler, just demonstrating pushing a list of cpu arrays to kvstore and pull the result.

Reply:
@apeforest had another discussion with @eric-haibin-lin , making it to a list of 4 CPU arrays will provide consistent result. If users has 1/2/3 GPUs, it will produce diffrent results compared to the value on tutorial, and may cause confusion. Also if only 1 GPU, it defeats the purpose of the tutorial.
I have added a note saying sum will only happen if value list is larger than 1.